lib/index.js

```diff
@@ -29,7 +32,7 @@ exports.default = function (sails) {
      * @param next
      */
     initialize: function initialize(next) {
-      next();
+      globalNext = next;
```

Comment:
In production mode, you are not executing globalNext.

To execute in production mode

```
if(process.env.NODE_ENV === 'development') {
globalNext = next;
}
else {
next();
}
```

I suggest you to rename globalNext to some meaningful name